.gitlab-ci.yml

```diff
@@ -49,7 +49,7 @@ check:lint:
   needs: []
   script:
     - >
-      nix-shell --arg ci true --run $'
+      nix develop .#ci --command bash -c $'
```

Comment:
How does this work? Does it use the `bash` from nixpkgs upstream? Can you specify `zsh` too? And is there no "default shell"?

Reply:
When running the `--command` argument, I was getting errors related to the syntax of the $'...'.

I'm not too sure why, but I read in the manpage for `nix develop` that they would use `bash -c` or a given shell to execute the command. I still don't know why, I have a feeling that the default `--command` argument has a bundled shell that does not have the same syntax as the shell code that was already written in the CI file.

I will look more into this, maybe we can make this more generic.

Reply:
It seems that `nix develop` as of my earlier research by default uses bash anyway, so it does not really change the outcome here. It just means its possible to specify a multiline command.